### ISSUE TITLE
use regex instead of string manipulations

### ISSUE THREAD

**Issue by jthompson18** [sarahmehmedi/comp330_twitterparser#5]
https://github.com/sarahmehmedi/comp330_twitterparser/blob/master/src/twitterparser/TwitterParser.java#L25 can be made more efficient with the use of regexs. Check out some of our classmate's repos for some examples. I'd recommend [Carter's](https://github.com/sarahmehmedi/comp330_twitterparser/blob/master/src/twitterparser/TwitterParser.java#L25)

**Comment by jthompson18** [comment]
@sarahmehmedi this will probably help you too https://github.com/twitter/twitter-text/tree/master/java/src/com/twitter

**Comment by sarahmehmedi** [comment]
@jthompson18 Thanks for the feedback! I did not know much about regex when I started this project that is why I was hesitant to use it. I do see how useful regexes are and will see if I will need to change my code to incorporate regexes! For this current project, I'm not sure I really want to change up my code unless more modifications are added and we would need to produce something out of it! Once again, I appreciate the time you took to help me make my code better and reliable!
